data/mods/Xedra_Evolved/snippets/dreams.json

```
@@ -5,26 +5,26 @@
     "text": [
       "You have bad dreams.",
       "You dreams are terrifying.",
-      "You dream some odd creatures are around you.",
-      "You feel like demon under your bed touched your hand.",
-      "You feel you just saw a boogeyman.",
-      "You start panicking for no reason.",
-      "You start to sweat enormously.",
-      "You don't know; would it be better to not sleep.",
+      "You dream of odd creatures circling you.",
+      "You feel like a demon under your bed just brushed your hand.",
+      "You heart races like you just came face to face with a bogeyman.",
+      "Panic seizes you suddenly with no discernible reason.",
+      "A torrent of cold sweat starts running down your back.",
+      "You think not sleeping could be the preferable option right now.",
       "You are afraid.",
-      "You feel a presence of terrifying creatures.",
-      "In your dream, an infinitely hot wind scorched your face.",
-      "In your dream, an enormous gravity compress your bones into dust.",
-      "In your dream, a strong winds throw you up into the sky.",
-      "In your dream, you are haunted by an inhuman giants.",
-      "In your dream, you are a toy for an unspeakable power.",
-      "In your dream, no one helps you.",
-      "in your dream, you feel like you were shot in the face from a rifle.",
-      "In your dream, you are no better than dead.",
-      "In your dream, it may be better to be actually dead.",
-      "In your dream, there is no one to ask from help.",
-      "In your dream, the world has lost all it's colors.",
-      "In your dream, the world is pitch black."
+      "You feel like terrifying creatures are surrounding you.",
+      "In your dream, an infinitely hot wind scorches your face until nothing but charred bone remains.",
+      "In your dream, enormous gravity crushes you against the ground until your bones turn to fine dust.",
+      "In your dream, a strong wind flies you up into the sky only to then let you plummet to your death.",
+      "In your dream, you are hunted by inhuman giants.",
+      "In your dream, you are the toy of an unspeakable power.",
+      "In your dream, no one is willing to help you.",
+      "In your dream, you feel like you were shot in the face with a rifle.",
+      "In your dream, you experience such agony than you are no better than dead.",
```

This sentence feels very off now. I am unsure what it was supposed to mean though. Maybe "than the dead?"

```suggestion
      "In your dream, you feel as if you are no better than the shambling dead.",
```